### DISCUSSION TITLE
播放的时候疯狂掉帧

### DISCUSSION THREAD

**Discussion by ethanqiii** [Paladinfeng/MiaoProject#182] [Q&A]
apple tv 16.1 使用商店下载的最新版本miao （2.0.1）在观看视频时无论什么视频，它都会在进度条消失后开始疯狂掉帧，所有设置均为软件下载后默认设置，更改hevc也没用，更改自动匹配帧率也没用，更改仅加载最高画质没用，更改加载杜比视界没用，更改播放链接优化没用，请教应该怎么办

**Comment by Paladinfeng** [comment]
1. 是否开了倍速播放？关掉
2. 设置里打开「过滤超过60帧的视频」

先尝试方法1，不行再试试方法2

**Reply by ethanqiii** [reply]
未开启倍速，电视和盒子都支持60帧过滤了画质会明显下降吗，感谢您解答

**Reply by Paladinfeng** [reply]
你先试试行不行，目前还没碰到过任何视频都会疯狂掉帧的情况

**Reply by ethanqiii** [reply]
超过60帧这个开了就不掉帧了

**Reply by aobozhou** [reply]
我开启了Apple TV系统-视频设置里的帧率匹配就会出现一样的问题。关掉Apple TV的帧率匹配后问题就会消失。
